.github/workflows/build.yaml

```diff
@@ -21,11 +21,11 @@ jobs:
         id: cache
         with:
           path: ~/local/rubies
-          key: ruby-2.7.1
+          key: ruby-2.7.2
       - uses: clupprich/ruby-build-action@master
         id: ruby
         with:
-          ruby-version: 2.7.1
+          ruby-version: 2.7.2
```

Comment:
I've passed this on to depfu and they are looking into adding this to their ruby version automation. That's mainly why this has not been upgraded yet.

Reply:
https://github.com/weg-li/weg-li/pull/273